docset/winserver2012r2-ps/failoverclusters/Get-ClusterVMMonitoredItem.md

```diff
@@ -138,7 +140,7 @@ Accept wildcard characters: False
 ```
 
 ### CommonParameters
-This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see about_CommonParameters (http://go.microsoft.com/fwlink/?LinkID=113216).
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

Comment:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [About CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```

Reply:
The actual article title is about_CommonParameters with lower case, so I believe we should leave it as is.

Reply:
Not the article that this link goes to. It is definitely upper case. Would you mind checking again?

Reply:
![image](https://user-images.githubusercontent.com/10958657/65811780-6a9da000-e1c6-11e9-90b8-fa129a64bdee.png)
All about articles start with lower case about.